src/main/java/frc/robot/Robot.java

```diff
@@ -22,6 +24,8 @@
 
   private RobotContainer m_robotContainer;
 
+  public static ColorSensor m_colorSensor = new ColorSensor();
```

Comment:
As part of the new command framework for this year, wpilib is pushing teams towards putting subsystems inside of the RobotContainer class. We should probably follow that.